packages/migration/migrations/performance/20230221121625-create-performance-views.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+export const up = async (db, client) => {}
```

Comment:
Should the up & down be in same files?

Reply:
Yea, this is a bit confusing. The reason for this is that the Hearth migration had to be the migration that creates the initial collections for the Performance database. The reason for this was that I didn't want the performance MongoDB user to have read access to hearth-dev. Instead we have the hearth user access to write data to the performance database. We could of course move this down migration to the Hearth migration that sets up the performance collections 🤔